apps/meteor/server/methods/createDirectMessage.js

```diff
@@ -89,6 +90,11 @@ export function createDirectMessage(usernames, userId, excludeSelf = false) {
 	if (excludeSelf && hasPermission(this.userId, 'view-room-administration')) {
 		options.subscriptionExtra = { open: true };
 	}
+	try {
+		callbacks.run('federation.beforeCreateDirectMessage', roomUsers);
+	} catch (error) {
+		throw new Meteor.Error(error?.message);
```

Comment:
any benefit in throwing a `Meteor.Error` here? just would like to understand if this try/catch is really needed

Reply:
The idea was to be able to send the right error to the front-end, I tried with `Error` only, but it looked like it was not working as expected, any idea?